workflows/mag.nf

```diff
@@ -736,6 +736,15 @@ workflow MAG {
         * DAS Tool: binning refinement
         */
 
+        ch_binning_results_bins = BINNING.out.bins
+                .map { meta, bins ->
+                    [meta + ['refined':'unrefined'], bins]
```

Comment:
I think as someone rather new to the pipeline code a meta entry: 'refined':'unrefined' is rather confusing. Maybe the entry should be named something like 'refinement' or 'refinement_tag' to make it more verbose.

Reply:
Agreed! Pr maybe 'bin-status' or if you prefer `un/refined` as a string? If you want to keep hte meta ID as `refined` it could be made boolean (but this will be more complicated for inserting into text